src/select-menu/src/SelectMenuContent.js

```diff
@@ -16,6 +16,7 @@ export default class SelectMenuContent extends PureComponent {
     hasTitle: PropTypes.bool,
     hasFilter: PropTypes.bool,
     listProps: PropTypes.shape(OptionsList.propTypes),
+    filterProps: PropTypes.shape({ placeholder: PropTypes.string }),
```

Comment:
Seeing `listProps` right next to this, it seems like `filterProps` might be redundant? Can/should we hook into the existing `listProps` prop?